settings/settings.go

```diff
@@ -29,13 +34,14 @@ func InitSettings() {
 	SMTPPassword = os.Getenv("SMTP_PASSWORD")
 
 	FromEmail = os.Getenv("FROM_EMAIL")
+	return nil
 }
 
 var EMAIL_INTERVAL time.Duration = 24 * time.Hour
```

Comment:
By the way, type declaration is redundant here, golang compiler will infer it :)